### PR TITLE
API search profile query now returns user lat and lon

### LINKED ISSUE #5774: Inclusion of lat and lon on API profile search query
The following issue on mapknitter requires this update to be able to return a profile user's location if they have added it on their profile
https://github.com/publiclab/mapknitter/issues/626

### PR DESCRIPTION
**Pull request by rarrunategu1**
Fixes #5774  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by rarrunategu1** [comment]
Yes!  It passed...well except for codeclimate upset about some tabbing on the code but the tabbing coincides with the rest of the tabbing in that particular file. 
image  
@jywarren I think I may have done something wrong with rebasing because now I don't have a special named branch to push instead I had to merge into my master at the end.  Will that cause an issue?

**Comment by gauravano** [comment]
Hi @rarrunategu1, no worries. You can reset your `master` branch by following - https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch. 

> Will that cause an issue?

Yes and No. It depends on how well you perform merging. Like, we'll do "Squash and merge" but if you do normal merging then you might see additional commits in next PR(considering you'll create a new branch from unsynced master). So, it's good to reset your branch and keep it synced with publiclab/plots2.

File: app/api/srch/search.rb [hunk 1]
```diff
@@ -103,7 +103,10 @@ class Search < Grape::API
               DocResult.new(
                 doc_type: 'USERS',
```
Comment (gauravano) [inline app/api/srch/search.rb]:
```suggestion
                  doc_type: 'USERS',
```

Comment (gauravano) [reply app/api/srch/search.rb]: I think codeclimate error will get resolved if you'll add two more spaces from 104-109. Would you like to try it? Although, the work here is done and we can merge it with the codeclimate error too. Thanks!

Comment (rarrunategu1) [reply app/api/srch/search.rb]: Oh.  I hadn't noticed that.  I'll make the change.

**Comment by rarrunategu1** [comment]
I believe this is ready for merging.  @jywarren @gauravano

**Comment by gauravano** [comment]
Merged :tada: :balloon: :+1:

**Comment by jywarren** [comment]
Awesome, and this should go live shortly as the production site gets republished. Thanks!

**Comment by rarrunategu1** [comment]
@gauravano @jywarren I just went back to check the code and unfortunately it doesn't look good with the alignment merged.  I don't think codeclimate made a good call on this one.  Its on line 104.  If I gave two additional spaces to everything then it wouldn't match up with the code on the remainder of the document.  What do you think?
https://github.com/publiclab/plots2/blob/master/app/api/srch/search.rb

**Comment by jywarren** [comment]
Line 104 does look 2 spaces too far indented, but it's not a huge deal. If
you want to submit a follow-up PR or include this in another PR that'd be
fine, or we can ignore for now! But much appreciated, it's great to have
the code look nice and tidy!

**Comment by gauravano** [comment]
Actually, when I left this - https://github.com/publiclab/plots2/pull/5780#discussion_r287616551 it was for all the parameters of the function. When it passed. I checked the code, but yeah, in-between +- snippet highlighting the spaces got ignored. It was weird that codeclimate didn't detect it, maybe it's programmed to detect space ambiguity for 1st parameter only, which I think is wrong behaviour. 

Anyway, I think opening an FTO for correcting this would be good. @rarrunategu1 would you like to open an FTO?

Thanks!

**Comment by rarrunategu1** [comment]
I would love to handle that.  I'll put it on my list of things to handle along with my other lower priority items.  

What is an FTO?  My apologies.. I'm not familiar with that term.

**Comment by jywarren** [comment]
It's a "first-timers-only" issue! You can read about it and learn how to
make one on this page: https://code.publiclab.org

**Pull request merged**